config/gatsby-remark-embedder/custom-yt-embedder.js

```diff
@@ -0,0 +1,59 @@
+const shouldTransform = url => {
+  const { host, pathname, searchParams } = new URL(url)
+
+  return (
+    host === 'youtu.be' ||
+    (['youtube.com', 'www.youtube.com'].includes(host) &&
+      pathname.includes('/watch') &&
+      Boolean(searchParams.get('v')))
+  )
+}
+
+const getTimeValueInSeconds = timeValue => {
+  if (Number(timeValue).toString() === timeValue) {
+    return timeValue
+  }
+
+  const {
+    2: hours = '0',
+    4: minutes = '0',
+    6: seconds = '0'
+  } = timeValue.match(/((\d*)h)?((\d*)m)?((\d*)s)?/)
```

Comment:
Minor nitpick, but you could use non-capturing groups (i.e. `/(?:(\d*)h)?(?:(\d*)m)?(?:(\d*)s)?/`) here to skip out on the otherwise useless group values and collect 1, 2, and 3 here.

Reply:
Done and worked the same as before!